stream-chat-android-client/src/test/java/io/getstream/chat/android/client/call/WrappedValueCall.kt

```diff
@@ -0,0 +1,11 @@
+package io.getstream.chat.android.client.call
+
+import io.getstream.chat.android.client.utils.Result
+
+internal class WrappedValueCall<T : Any>(private val value: T) : Call<T> {
```

Comment:
Note: this is the same as the `TestCall` class in one of our other modules. Let's leave it as-is for this PR though, I'll move them into a common test utilities module after this is merged (along with some other test stuff).

Reply:
Sounds good. Maybe we need not core module, but some test-core module. I think If we start to move everything in new core module it soon will became a garbage module

Reply:
Yeah, we need a separate test module, because we only want to include that as a `testImplementation` dependency. It was just wasn't possible before, because it requires us to move some stuff to `core` first, so that the test module can depend on it.